Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/GoogleAPIVersion.kt

```diff
@@ -22,8 +22,8 @@ class GoogleAPIVersion @Inject constructor() {
             val currentVersion = InternalExposureNotificationClient.getVersion()
             currentVersion >= compareVersion
         } catch (apiException: ApiException) {
-            if (apiException.statusCode == CommonStatusCodes.API_NOT_CONNECTED) false
-            else throw apiException
+            if (apiException.statusCode != CommonStatusCodes.API_NOT_CONNECTED) throw apiException
+            else return false
```

Comment:
no "else" needed

Reply:
removing else resulted in Type mismatch error. But will check for any alternate implementation to remove that

Reply:
ok then